### PR TITLE
Revert "chore: update `no-response` workflow"

### PR DESCRIPTION
**Pull request by machour**
Reverts remix-run/examples#217

### CODE REVIEW

**Review by mcansh — APPROVED** [review]
if we want these to not get automatically marked as stale, we need the `-1` trick https://github.com/actions/stale#days-before-stale

File: .github/workflows/no-response.yml [hunk 1]
```diff
@@ -17,25 +17,16 @@ jobs:
       - name: 🥺 Handle Ghosting
         uses: actions/stale@v8
         with:
+          days-before-close: 21
```
Comment (machour) [inline .github/workflows/no-response.yml:20]:
```suggestion
          days-before-close: 21
          # don't automatically mark issues/PRs as stale
          days-before-stale: -1
```

**Comment by machour** [comment]
@mcansh nice catch 🙏🏼 fixed

**Comment by machour** [comment]
By the way, I left 21 on purpose here, since it's the examples repository, and we want people to contribute at their own pace

**Pull request merged**